bn_mp_decr.c

```diff
@@ -0,0 +1,46 @@
+#include "tommath_private.h"
+#ifdef BN_MP_DECR_C
+/* LibTomMath, multiple-precision integer library -- Tom St Denis
+ *
+ * LibTomMath is a library that provides multiple-precision
+ * integer arithmetic as well as number theoretic functionality.
+ *
+ * The library was designed directly after the MPI library by
+ * Michael Fromberger but has been written from scratch with
+ * additional optimizations in place.
+ *
+ * SPDX-License-Identifier: Unlicense
+ */
+
+/* Decrement "a" by one like "a--". Changes input! */
+#ifdef LTM_USE_EXTRA_FUNCTIONS
```

Comment:
IMO this macro can be removed

Reply:
It changes the input, not a very common thing in LibTomMath, so I was a bit cautious here.

But if that (remove the macro for this function and this function only) is not what you meant drop me a note please.